src/Application.vala

```diff
@@ -21,6 +21,7 @@
 
 namespace Akira {
 	public Akira.Services.Settings settings;
+  public Akira.Services.EventBus event_bus;
```

Comment:
Check your indentation here

Reply:
Hi @Philip-Scott, just another question: do I have to check and correct also the wrong indentation I did on other files? I'm doing it right now, since every file I created has 2 spaces instead of 4 for indenting!

Reply:
Another thing, pardon me if I'm bothering you with very little issues and questions, but I want to be as compliant as possible to the rules and I also want this PR to be merged (looks like an Odyssey to me :) ).

I added to the project (**not** to the repo, just in my folder) an `.editorconfig` file, the one provided in the ElementaryOS guidelines. 

The only problem is that the *license* is not indented correctly, instead of being like this: 

```
/*
* Copyright (c) 2018 Alecaddd (http://alecaddd.com)
*
* This program is free software; you can redistribute it and/or
```

It is automatically formatted this way (please note the leading space between the `*`):
```
/*
 * Copyright (c) 2018 Alecaddd (http://alecaddd.com)
 *
 * This program is free software; you can redistribute it and/or
```
I don't think this is something lika an *issue*, but it looks actually different from the code present in the repo. 

What do you think?

Reply:
We are going to need to standardize the indentation here :) Alex used Tabs before although spaces is what should be used. You should just indent the code as it's on the file you're currently editing, and use 4 spaces if you are making a new file as that's the official recommendation. So if the file currently has tabs, use tabs :) 

We will go though the project soon and make it all use the same indentation format

Reply:
Hi all, 

In the end I was able to format the code correctly, I used the correct indentation for the license and the code.